src/audio-worklet/free-queue/src/interface/C/free_queue.h

```diff
@@ -0,0 +1,206 @@
+#ifndef FREE_QUEUE_C
+#define FREE_QUEUE_C
+
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdatomic.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef EMSCRIPTEN_KEEPALIVE
+#define EMSCRIPTEN_KEEPALIVE __attribute__((used))
+#endif
+
+/**
+ * C Struct to represent FreeQueue
+*/
+struct FreeQueue {
+    size_t bufferLength;
+    size_t channelCount;
+    atomic_uint* state;
+    float** channels;    
+};
+
+enum FQState {
```

Comment:
Be descriptive: `FreeQueueState` is better than `FQState`.

You write once, but will read this a lot moving forward. Descriptive naming can be huge a time saver.